### ISSUE TITLE
[Suggestion] Add a framed lever!

### ISSUE THREAD

**Issue by katubug** [EpicSquid/Blockcraftery#45]
Hello! Firstly, thank you so much for filling the void in customizable shaped blocks. It's been a staple in all my modpacks for ages.

I think a customizable lever would be an excellent addition, if it's something that would be possible.

Thanks for your time!

**Comment by EpicSquid** [comment]
What's ur desire for a framed lever? Does only the base change? Or both the base and the handle?

**Comment by katubug** [comment]
Ideally, the base and handle would both change, but just the base would also be fine. It's just that unless you're building with cobblestone, they don't really match anything, haha.

**Comment by EpicSquid** [comment]
I want to add a way to edit the base easily, the handle is a bit trickier, since blockcraftery doesn't currently have support for 2 textured blocks.

**Comment by katubug** [comment]
Yeah, that's understandable. I know that Storage Drawers has a Framed Drawer option that allows you to add three different textures, but those aren't applied by right clicking like in Blockcraftery. You need to assemble them in a framing table.

Still, maybe you could see how they do it?
